### DISCUSSION TITLE
What's taking up all the 1000 lines in the main file?

### DISCUSSION THREAD

**Discussion by TechStudent10** [ThioJoe/YT-Spammer-Purge#412] [General]
Just curious, because even though you've already separated the functions into their own files, it still has like 1.1K lines.

I'm just wondering.

**Comment by ThioJoe** [comment]
Bruh it's literally open source. But it's mostly navigation and supporting all the possible prompts and options and stuff.

**Reply by TechStudent10** [reply]
> Bruh it's literally open source.

Yes I know but I have no idea how the Google API works so it's kind of confusing.

> But it's mostly navigation and supporting all the possible prompts and options and stuff.

Ah okay.
